csharp-intermediate-topics/ReadOnlyModifierInCSharp/ReadOnlyTests/ReadOnlyTests.cs

```diff
@@ -0,0 +1,26 @@
+using ReadOnlyModifierInCSharp;
+
+namespace ReadOnlyTests
+{
+    public class ReadOnlyTests
+    {
+        [Fact]
+        public void GivenCircleWhenRadiusSetInCtorThenAreaCalculatedUsingRadius()
+        {
+            var circle = new Circle(7.5);
+            var expectedArea = 176.71;
+
+            Assert.Equal(expectedArea, Math.Round(circle.Area, 2));
```

Comment:
Rather than rounding, we should use one of the Assert.Equal double overloads specifying either a precision (int value) or a tolerance (double value). So the equivalent to the above code would be:
`Assert.Equal(expectedArea, circle.Area, 2);`
With a tolerance value we would write something like:
`Assert.Equal(expectedArea, circle.Area, 1e-2);`